cmake/Options/BuildOptions.cmake

```diff
@@ -53,6 +53,10 @@ if (NOT TILEDB_VCPKG)
   message(FATAL_ERROR "Disabling TILEDB_VCPKG is not supported. To disable automatically downloading vcpkg, enable the TILEDB_DISABLE_AUTO_VCPKG option, or set ENV{TILEDB_DISABLE_AUTO_VCPKG} to any value.")
 endif()
 
+if (TILEDB_HDFS)
+  message(DEPRECATION "The HDFS storage backend is deprecated and untested. It will be removed in TileDB 2.28.")
```

Comment:
Let's mention that we only test that it builds.

Reply:
Updated.